ow_sim_tests/test/arm_check.py

```diff
@@ -89,7 +90,7 @@ def _test_activity(self, service_name, service_type,
       norm_joints_state_array = np.array(
           [self._normalize_angle(e) for e in joints_state])
       joints_abs_diff_array = np.abs(
-          joints_goal_array - norm_joints_state_array)
+          norm_joints_goal_array - norm_joints_state_array)
```

Comment:
I think it's safer to normalize to {-pi, pi} after this subtraction happens. If you normalize both values before, there's a small chance your difference could end up being close to 2*pi, much greater than the tolerance.

Reply:
That makes sense. I wanted to prove it to myself with a test. So setting `l_hand_yaw` to exactly pi for `arm_unstowed` in **lander.rsdf**, I did confirm this can happen, and it happens only half the time, which makes sense. I would expect the error depends on whether final joint state is higher/lower than joint goal.
```[rosout][INFO] 2021-11-23 11:32:58,055: 
      current joints state: [ 0.    1.57 -1.57 -0.   -3.14  0.01]
      diff joints state: [0.   0.   0.   0.   6.28 0.01]
      maximum diff between expected/current: 6.28
[rosout][INFO] 2021-11-23 11:32:58,257: operation /arm/unstow took 30s, time-limit is 30s
```
I get the same fails half of the time result when using -pi.

As of commit 3a22af it now normalizes after taking the difference between the angles.

Reply:
@mogumbo You comment is right but doesn't apply to the case of the test. We are checking for equal angle values so when both values are normalized and they are equal they wouldn't exceed the {-pi, pi} otherwise they are not equal to start with. I reverted the changes made to this code block to its original implementation to keep it simple.